### ISSUE TITLE
Doesn't work with polymer 2

### ISSUE THREAD

**Issue by anandanand84** [PolymerElements/paper-icon-button#116]
### Description

paper-icon-button shows no icon with polymer 2.0.
### Expected outcome

An Icon is displayed

### Actual outcome

No icon is displayed

### Live Demo
https://github.com/anandanand84/polymer-icon-button-test

### Steps to reproduce

1. Create a new application using polymer init
2. Import polymer-icon-button and add a paper-icon-button with icon=menu
3. There wont be any icons visible.

**Comment by web-padawan** [comment]
You need to import corresponding icons separately, e. g. `iron-icons` (though I recommend creating your own iconset with only required icons instead).

**Comment by anandanand84** [comment]
@web-padawan thank you. It does work after import iron-icons. But this line in the docs is misleading 

```
paper-icon-button includes a default icon set. Use icon to specify which icon from the icon set to use.
```

**Comment by notwaldorf** [comment]
Ah yes, there's another issue about removing this line. Hold on, sending a PR

**Comment by notwaldorf** [comment]
Closed in #117
